ansible/roles/omeka/tasks/main.yml

```diff
@@ -57,6 +53,22 @@
      - Search-By-Metadata-1.2.zip
    when: omeka_exists.stat.exists == false
 
+ - name: Set the permissions on the Omeka directory
+   file: path=/var/www/omeka/files owner=www-data group=www-data recurse=yes
+   when: omeka_exists.stat.exists == false
+
+ - name: Set the permissions on the Omeka directory
+   file: path=/var/www/omeka/admin owner=www-data group=www-data recurse=yes
```

Comment:
Why does the admin folder need `www-data` writeable? I can't see anything in the documentation about this.